CONTRIBUTING.md

```diff
@@ -173,12 +173,19 @@ source ./env/bin/activate
 trial tests.rest.admin.test_room tests.handlers.test_admin.ExfiltrateData.test_invite
 ```
 
-If your tests fail, you may wish to look at the logs:
+If your tests fail, you may wish to look at the logs (the default log level is `ERROR`):
 
 ```sh
 less _trial_temp/test.log
```

Comment:
It would be interesting to add a note to the `trial tests` output when the tests fail to go checkout the `_trial_temp/test.log` file. I wonder if we can hook in at all 🤔

Reply:
It looks possible if we defined our own reporter but then that requires adding `--reporter=ourcustomreporter` to the test command.

Anyway, another possible PR ⏩